### PR TITLE
Rename internal packages

### PR DESCRIPTION
**Pull request by mpraski**
- Rename internal packages to "vimeda" so that `go get` can resolve them
- Update readme

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This project adheres to the Contributor Covenant [code of conduct](CODE_OF_CONDU
 By participating, you are expected to uphold this code.
 Please report unacceptable behavior to [engineering@hellofresh.com](mailto:engineering@hellofresh.com).
```
Comment (FelipeUmpierre) [inline CONTRIBUTING.md:14]: Should we keep their email here?

Comment (mpraski) [reply CONTRIBUTING.md:14]: Right, maybe not

Comment (italolelis) [reply CONTRIBUTING.md]: This is a fork, we should keep everything the same... including the namespaces

**Comment by mpraski** [comment]
Closing, will use the `replace` directive to use the correct fork.

**Pull request closed without merging**